kdump-utils.spec

```diff
@@ -0,0 +1,136 @@
+# kdump-utils has no debug source
+%global debug_package %{nil}
+Name: kdump-utils
+Version: 1.0.42
```

Comment:
@coiby Does the version number in the spec need to be changed to the latest?

Reply:
Thanks for the reminding! I've updated the spec file to the latest version in #25. Sorry I have to open a new PR when this PR gets closed automatically after I rename my own branch by mistake.